proposals/003-installer-guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Introduction
+
+The Haskell Foundation would like see a central Haskell download page in which the Haskell curious
+could easily download and run a installer to provide a standard Haskell environment that supports
+the principal build tools (`cabal-install` and `stack`) on the main platforms (Linux, macOS and
+Windows). All Haskell introductory material (text books, blog posts, etc.) should be able to point
+to the Haskell Foundation download page and work on the assumption that it will yield their
```

Comment:
Will this be a separate resource to https://www.haskell.org/downloads/ ?

Reply:
I hope not!  It's clear to me that haskell.org should remain the contact point for getting started with Haskell.  It's where beginners will look, after all, whether they are told to or not.

Reply:
It had been communicated to me that there very well might be a separate resource if haskell.org doesn't "comply". How much of that was really true, I don't know.

I personally hope that HF creates their own download page and that haskell.org stays independent. But that's just my opinion.